smdebug/core/reader.py

```diff
@@ -54,3 +57,23 @@ def read_tensors(self, check="minimal"):
         if check is True:
             check = "minimal"
         return self._reader.read_tensors(check=check)
+
+    # Read the events that match the given regex list. If Regex list is not specified all events are read.
+    def read_events(self, check="minimal", regex_list=None):
```

Comment:
please add docstring instead of comment on top, so the function is properly documented. Is not clear what the check "minimal argument is doing. I know the code was there before, but since you are now modifying it is a good opportunity to improve it if only slightly.

Reply:
Added